### ISSUE TITLE
Concurrent multiple task backends

### ISSUE THREAD

**Issue by ripper234** [kugelblitz-one/Fredo-Public#89]
(Apologies if this is a duplicate)

As a user, I would like to configure some #hashtags to go to different backends.

E.g. I would like:

1. My default list to go to Workflowy
2. Any #lists I configure would go either to Workflowy or to Trello (any number of different Trello boards), depending on the specific link I paste.

From what I understand, most users use more than one task management system. Definitely if Email is included.

Can we find other concrete users who would use this feature in this way?

**Comment by assafza** [comment]
After discussing with @liatzionimorad we will put this on hold based on the following reasons:
1. We still don't know which platforms we are adding next (this is a work in progress)
2. Once we will have users we will be able to validate this specific need. so the focus would go to acquiring more active users to tell us their problems and needs first.
putting this in low priority for now and we will revisit this in a month time, hopefully after we have more users.

**Comment by assafza** [comment]
@JerryVDP Can you please create design for this feature first so we are sure we can handle it and support

**Comment by JerryVDP** [comment]
@assafza @liatzionimorad 
I need a UI/UX/Flow diagram for everything related to this.
For example, what will happen if a user tries to remap a hashtag that has already been mapped? should we just override the original one or show a message?

**Comment by assafza** [comment]
@JerryVDP @liatzionimorad 
I made a simple MVP for testing this idea:
https://miro.com/app/board/o9J_lttkfEA=/?moveToWidget=3458764530219535007&cot=14

Requirements:
1. User should be able to move between platforms
2. User should be able to have his platform setup saved (listmaps, default, etc)
3. We should make sure mixpanel analytics events is not damaged when user switch between platforms, meaning it will change the platform property 
4. Added analytics events to this action:  
5. Show the user a list of the existing platforms, show which of them is "active" meaning has an active default board/list.
